core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -2672,6 +2673,46 @@ public void propagatePersisted() throws Exception {
             GetStatusContext.defaults()).getPersistenceState());
   }
 
+//  @Test
```

Comment:
What about this test?

Reply:
I was trying to write a test which reproduced the issue consistently, but this one only reproduced it every 1/25 times or so...I can try adding more to make sure that it does reproduce the issue and that this change fixes it, but it might take me a bit longer - or we can just go without this test.

What do you think?

Reply:
even with a longer path is failed to reproduce. To writing the actual test that fails (deadlocks) in the earlier code and not in the current code might take me a while to write.